### DISCUSSION TITLE
Faraday Connection Failed

### DISCUSSION THREAD

**Discussion by gummipunkt** [mastodon/mastodon#22878] [General]
### Steps to reproduce the problem

Job: Scheduler::IndexingScheduler
Error: Faraday::ConnectionFailed: Connection refused - Connection refused - connect(2) for "localhost" port 9200 (localhost:9200)

### Expected behaviour

Should work

### Actual behaviour

Connection Failed

### Specifications

Mastodon 4.0.2

**Comment by ineffyble** [comment]
This suggests you have ElasticSearch configured but it is not running.
